molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/impl/SnpEffAnnotator.java

```diff
@@ -111,7 +119,7 @@ public static String getGeneNameFromEntity(Entity entity)
 		}
 		if (geneSymbol == null)
 		{
-			String annField = entity.getString(VcfRepository.getInfoPrefix() + "ANN");
+			String annField = entity.getString("ANN");
```

Comment:
Annotators are not specific for VCF, the CmdLineAnnotator and VCFUtils should handle the prefix. The annotator should return a logical attributename